ceph.spec.in

```diff
@@ -1086,36 +1081,19 @@ rm -rf %{buildroot}
 
 %post base
 /sbin/ldconfig
-%if 0%{?suse_version}
-%fillup_only
-if [ $1 -eq 1 ] ; then
-/usr/bin/systemctl preset ceph-disk@\*.service ceph.target >/dev/null 2>&1 || :
-fi
-%endif
-%if 0%{?fedora} || 0%{?rhel}
-%systemd_post ceph-disk@\*.service ceph.target
-%endif
 if [ $1 -eq 1 ] ; then
 /usr/bin/systemctl start ceph.target >/dev/null 2>&1 || :
 fi
 
-%preun base
-%if 0%{?suse_version}
-%service_del_preun ceph-disk@\*.service ceph.target
-%endif
-%if 0%{?fedora} || 0%{?rhel}
-%systemd_preun ceph-disk@\*.service ceph.target
```

Comment:
The same here, we should not remove these lines for ceph.target.

Reply:
excellent catch, let me fix this